CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Changelog
 
+## 16.2.0 (08/26/24)
+
+### NLA Support
+
+Teleport will support Network Level Authentication (NLA) when connecting to
```

Comment:
We should remove the "will" from these as they are no longer future-looking statements.